master/reference/calicoctl/commands/label.md

```diff
@@ -0,0 +1,153 @@
+---
+title: calicoctl label
+canonical_url: 'https://docs.projectcalico.org/v3.5/reference/calicoctl/commands/apply'
```

Comment:
canonical URL isn't set correctly IIUC

Reply:
I'm not sure what the correct value is supposed to be...definitely `label` at the end of the path instead of `apply`. But what about the version path component? leave as v3.5?